test/integration/basic_test.go

```diff
@@ -169,12 +169,12 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`([^"]*) should equal`, m.commandReturnShouldEqualContent)
 	s.Step(`([^"]*) should be empty`, m.commandReturnShouldBeEmpty)
 
-	s.BeforeScenario(func(interface{}) {
+	s.BeforeSuite(func() {
```

Comment:
So what does this mean in terms of having multiple feature files? When is this run? 

@gbraad How is this going to work when we start having multiple feature files? Correct me from wrong, but we can have multiple test files in each of which we define _func FeatureContext(s *godog.Suite)_. As long as the file is annotated with _'// +build integration'_ it will add a new suite? So running the setup and teardown before/after suite would be ok for now. Would this be correct? 

@agajdosi Could you investigate this please?

If this is the case, should we not extract all these step definitions (or at least the ones which we want to share) and put them into a separate file _common_steps.go_ with a function _addCommonSteps(s *godog.Suite)_?